### PR TITLE
Fix link to Getting Started page

### PR DESCRIPTION
**Pull request by pschloss**
The current link goes to a page that doesn't exist

### CODE REVIEW

**Review by cboettig — APPROVED** [review]

**Comment by cboettig** [comment]
thanks! 

aside, but I wonder if our readme should now mention new alternative options available.  In particular, huggingface.co/datasets supports git LFS without the restrictions/problems we mention regarding LFS on GitHub.

**Comment by cboettig** [comment]
@tanho63 I'll let you merge this when ready, I think the failed checks look like auth-related things that should resolve on merge, yes?

**Comment by tanho63** [comment]
this LGTM. check failures look unrelated to PR contents, probably bad gha setup. will fix in separate PR

**Pull request merged**
